packages/web/src/scss/components/Pagination/index.html

```diff
@@ -152,4 +152,52 @@
   </ul>
 </nav>
 
+<nav role="navigation" aria-label="Pagination Navigation" class="mb-800 text-center">
```

Comment:
Hmm, what's the `Navigation` class for? 🤔 It's present in all examples… Seems to me as an error.

Reply:
First of all I'll change the order of the HTML attributes: `class` – `role` – `aria-label` (in case we don't have any standard for HTML attributes order).